project/CIRelease.scala

```diff
@@ -0,0 +1,84 @@
+package com.geirsson
```

Comment:
this file is copy-pasted from https://github.com/olafurpg/sbt-ci-release, and adapted to make it work with Concourse (the original one only works with Travis)

I may publish it as a plugin at some point